content/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -138,6 +162,53 @@ the possible following fields:
 
 `dvc.yaml` files also support `# comments`.
 
+### dvc.lock file
+
+The purpose of `dvc.lock` is to lock the exact contents of all files involved in
+the pipeline(s) at the time that `dvc.yaml` was last updated.
+
+Here's an example `dvc.lock` based on the one in [`dvc.yaml`](#dvcyaml-file)
+above:
+
+```yaml
+stages:
+  features:
+    cmd: jupyter nbconvert --execute featurize.ipynb
+    deps:
+      - path: data/clean
+        md5: d8b874c5fa18c32b2d67f73606a1be60
+    params:
+      params.yaml:
+        levels.no: 5
+    outs:
+      - path: features
+        md5: 2119f7661d49546288b73b5730d76485
+      - path: performance.json
+        md5: ea46c1139d771bfeba7942d1fbb5981e
+      - path: logs.csv
+        md5: f99aac37e383b422adc76f5f1fb45004
+```
+
+It replicates the structure of `dvc.yaml`, but with some key differences and
+additions to <abbr>dependency</abbr> and <abbr>output</abbr> fields:
+
+- `deps`: Each item of the list becomes an object with 2 properties:
+  - `path`: Same as the `deps` value from `dvc.yaml`
+  - `md5`: The hash value of the file
+  - `etag`: Strong ETag response header (only HTTP <abbr>external
+    dependencies</abbr>)
+- `params`: List of [parameters files](/doc/command-reference/params#examples)
+  (typically only `params.yaml`), as well as sub-lists of the specific
+  parameters tracked by this stage (per params file), and their latest value.
+- `outs`: Each item of the list becomes an object with 2 properties:
+
+  - `path`: Same as the `deps` value from `dvc.yaml`
+  - `md5`: The hash value of the file
+
+  This includes any [metrics files](/doc/command-reference/metrics) or
```

Comment:
not sure I understand this tbh

Reply:
This no longer applies (it's been rewritten anyway) per https://github.com/iterative/dvc.org/pull/1448#discussion_r443045904. Resolving